app.py

```diff
@@ -22,7 +30,17 @@ def beep(repeat):
     body = sel.find_element_by_tag_name('body')
     if body.get_attribute('class') == 'queue challenge':
         queue = True
-        beep(5)
+        say("ps5 queue is active")
+    if body.get_attribute('class') == 'softblock':
```

Comment:
Should this be elif? There wouldn't be a case where the body class name is both.

Reply:
shit good catch didn't notice that